lib/pymedphys/tests/dicom/test_coords.py

```diff
@@ -88,3 +91,41 @@ def test_extract_iec_fixed_xyz():
 @pytest.mark.pydicom
 def test_extract_dicom_patient_xyz():
     run_xyz_function_tests("DICOM")
+
+
+@pytest.mark.pydicom
+def test_coords_in_datasets_are_equal():
+    test_dicom_dict = {
+        "ImagePositionPatient": [-1.0, -1.0, -1.0],
+        "ImageOrientationPatient": [1, 0, 0, 0, 1, 0],
+        "BitsAllocated": 32,
+        "Rows": 3,
+        "Columns": 3,
+        "PixelRepresentation": 0,
+        "SamplesPerPixel": 1,
+        "PhotometricInterpretation": "MONOCHROME2",
+        "PixelSpacing": [1.0, 1.0],
+        "GridFrameOffsetVector": [0.0, 1.0, 2.0],
+        "PixelData": np.ones((3, 3, 3)).tobytes(),
```

Comment:
Likely not an issue here, but I personally like to use test data that has axes with differing size and asymmetric values. ...can pick up weird things sometimes...

Reply:
See new test data: https://github.com/pymedphys/pymedphys/pull/1405/files#diff-31b2aa7a89df85c2df255a328476c05b263630c620158d2d429635578f80b655R105